jcapi-systemusers.go

```diff
@@ -149,6 +160,16 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) e
 			return err
 		}
 	}
+
+	if _, exists := fields["attributes"]; exists {
+		attributes := fields["attributes"].([]interface{})
```

Comment:
this can avoid the double-lookup and instead become something like
```
if attrs, exists := fields["attributes"]; exists {
    attributes := attrs.([]interface{})
}
```